### ISSUE TITLE
Checkbox with tooltip does not show focus treatment

### ISSUE THREAD

**Issue by MrBunny194** [angular/components#15294]
#### What is the expected behavior?
The checkbox doesn't have focus(checkbox has matTooltip) when pressing tab
![screenshot_3](https://user-images.githubusercontent.com/29162621/53337302-11a91900-3912-11e9-987d-5948dd891547.png)

#### What is the current behavior?
Set focus on the checkbox (checkbox has matTooltip) when pressing tab.
![image](https://user-images.githubusercontent.com/29162621/53339908-add61e80-3918-11e9-8029-a0542e85509e.png)

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Material 7.2.0 - not working
Material 6.4.7 - working

**Comment by jelbourn** [comment]
Confirmed that the checkbox does receive focus but does not render its focus treatment when it has a `matTooltip` applied.

**Comment by jerome-nelson** [comment]
Seems to be fixed in ^8.0.1

**Comment by CaselIT** [comment]
still present as of 8.2.3

link to stackblitz

https://stackblitz.com/edit/components-issue-pethx5
https://components-issue-pethx5.stackblitz.io

**Comment by Panda-313** [comment]
I've started working on that so if it's possible, please add "in progress" label :wink:

**Comment by saimounikaperi** [comment]
Hey, Is anyone working on this? If not, I will take it up

**Comment by jelbourn** [comment]
@viniciusschuelter thanks for the note, it does indeed seem to be fixed
